common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.exception;
+
+public class ErrorStrings {
+    /**
+     * There are multiple cache entries found, the sdk cannot pick the correct access token
+     * or refresh token from the cache. Likely it's a bug in the sdk when caching tokens or authority
+     * is not proviced in the silent request and multiple tokens were found.
+     */
+    public static final String MULTIPLE_MATCHING_TOKENS_DETECTED = "multiple_matching_tokens_detected";
```

Comment:
One question, as the ErrorString variables are all String type. What if the dev passed in invalid error String, which is not defined in the ErrorStrings class? Why enum is not preferred?

Reply:
enums leave our partners comparing large formatted strings, i wanted strings to simplify their life. There will be more error strings exposed as we push for showing more internal errors. I do not have a strong preference per se though.